### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by kkolade**
### Implemented mobile menu requirement in the project by adding JavaScript for interactivity
- Added javascript file app.js to the ./js folder
- Rename HTML elements relevant to interactivity implementation to have semantic meaning
- Selected relevant HTML elements in JS file and added interactivity
- Style the selected elements to reflect the expected result

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi Team👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkolade** [comment]
@brytebee thanks for the wonderful feedback, the error is fixed
- The hamburger icon was removed from the desktop view
- The toggle between the bars icon and the times icon now persists prior to now it stops toggling after 3 clicks

File: index.html [hunk 1]
```diff
@@ -24,20 +24,19 @@
...
             </li>
             <li class="main-nav__list-item">
-              <a href="" class="main-nav__link">About</a>
+              <a href="#about" class="main-nav__link">About</a>
             </li>
             <li class="main-nav__list-item">
-              <a href="" class="main-nav__link">Contact</a>
+              <a href="#contact" class="main-nav__link">Contact</a>
             </li>
           </ul>
```
Comment (ShahierNashaat) [inline index.html:41]: - [ ] [OPTIONAL] Kindly notice that the mobile menu should disappear when the user clicks a menu item.

**Review by ShahierNashaat — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**